docs/source/executors/executors.rst

```diff
@@ -67,6 +71,7 @@ SSH Executor
         username: clown
         client_keys:
           - /opt/tardis/ssh/tardis
+        on_disconnect_retry: true
```

Comment:
Should be `True`.

Reply:
Not exactly a hill I want to die on, but [lowercase `true`/`false` is the canonical YAML form](https://yaml.org/spec/1.2.2/#10212-boolean). I realise that I did it "wrong" myself in the prose, so take your pick which one I should change.